easybuild/easyconfigs/d/dagitty/dagitty-3.0-foss-2021b.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'RPackage'
+
+name = 'dagitty'
+version = '3.0'
+
+homepage = 'https://github.com/jtextor/dagitty'
```

Comment:
@deniskristak Since `dagitty` is available through CRAN (see https://cran.r-project.org/web/packages/dagitty), we should just add it as an extra extension in the `R` easyconfig instead, see for example https://github.com/easybuilders/easybuild-easyconfigs/pull/14750/files).

Please also check the other ones, everything available through CRAN should be added to R itself (`CMAverse` is not, so that should remain a stand-alone easyconfig)

Reply:
@deniskristak for clarification, `dagitty` used to be a stand-alone easyconfig because it depended on V8, but now V8 is also included in the main `R` easyconfig.